Source/WebGPU/WGSL/TypeDeclarations.rb

```diff
@@ -11,3 +11,17 @@
     [T < Float, C, R].(Matrix[T, C, R], Vector[T, C]) => Vector[T, R],
     [T < Float, C, R].(Vector[T, R], Matrix[T, C, R]) => Vector[T, C],
 }
+
+operator :textureSample, {
```

Comment:
Can we document the missing `textureSample` functions with a FIXME?

Reply:
I added a link to a bugzilla bug that lists all the missing overloads